python-sdk/langwatch/tracer.py

```diff
@@ -105,6 +105,7 @@ class ContextSpan:
     model: Optional[str] = None
     params: Optional[SpanParams] = None
     metrics: Optional[SpanMetrics] = None
+    max_string_length: Optional[int] = 5000
```

Comment:
why do you have this defined at ContextSpan as well if it's always used from trace only?